src/main/kotlin/com/angorasix/projects/management/core/presentation/handler/ProjectsManagementHandler.kt

```diff
@@ -267,7 +267,8 @@ private fun ProjectManagementDto.resolveHypermedia(
     add(getByProjectIdAffordanceLink)
 
     // edit ProjectManagement
-    if (simpleContributor != null && simpleContributor.isProjectAdmin) {
```

Comment:
created Trello task to implement this logic correctly:
https://trello.com/c/bmSXmaFq/18-replicate-admins-channges-in-mgmtcore

(so, no need to change anything here, we'll work on that as a separate task)